### PR TITLE
Add baserun to beebot

### PR DESCRIPTION
**Pull request by adam-baserun**
To get up and running:
1. Make an account with [Baserun](https://baserun.ai) and then create an API key at [https://baserun.ai/settings](https://baserun.ai/settings) and add it to your .env file
2. `poetry run pytest --baserun` will upload test results to Baserun and print out a link to the test result page.

Grab the link to the baserun results page after the tests complete:

Screenshot 2023-08-01 at 6 26 14 PM

And then view and compare the results in a number of ways:

Screenshot 2023-08-01 at 6 26 14 PM

Screenshot 2023-08-01 at 6 26 14 PM

Screenshot 2023-08-01 at 6 26 14 PM

Screenshot 2023-08-01 at 6 26 14 PM

Screenshot 2023-08-01 at 6 26 14 PM

### CODE REVIEW

File: yoyo.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
 sources = migrations
-database = postgresql://postgres:postgres@localhost/postgres
+database = %(DATABASE_URL)s
```
Comment (adam-baserun) [inline yoyo.ini:3]: Happy to pull this one out into its own PR if you prefer. Was hitting an issue in my own setup because I was using a different postgres user

File: tests/conftest.py [hunk 2]
```diff
@@ -10,6 +10,10 @@
 from beebot.body import Body
 
 
+def pytest_configure():
```
Comment (adam-baserun) [inline tests/conftest.py:13]: This change ensures that the environment variables are loaded before the baserun pytest plugin starts up

File: beebot/body/body.py [hunk 3]
```diff
@@ -116,6 +117,13 @@ def cycle(self) -> Memory:
         self.execute(decision=self.decide())
 
         complete_memory = self.current_memory_chain.finish()
+
+        baserun.log("CycleComplete", payload={
```
Comment (adam-baserun) [inline beebot/body/body.py:121]: This is a noop if baserun is not initialized. We are automatically tracking OpenAI calls but I remember hearing that it's also useful to track aggregated info after every cycle, so added this as one option

**Review by erik-megarad — APPROVED** [review]
